backend/de.metas.report/de.metas.report.jasper.client/src/main/java/de/metas/report/jasper/client/interceptor/AD_Process.java

```diff
@@ -1,5 +1,11 @@
 package de.metas.report.jasper.client.interceptor;
```

Comment:
pls move this class to a more "generic" module&package..or if that's not feasilbe, another possibility might be to create one for jasper and one for postgrest and write comments to mae sure future devs don't change them to interfere with each other